employee.rb

```diff
@@ -0,0 +1,22 @@
+
+class Employee
+  attr_accessor :name, :salary, :performance_review, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @performance_review = []
+    @satisfactory = []
+  end
+
+  def pay_raise(employee)
```

Comment:
At Cummins, 7% percent was the highest you could get.  0%, 3%, 5% or 7%.
Those were the annual merit increases that were avialable.